packages/guesstimator/formatter/formatters/lib.js

```diff
@@ -57,3 +57,39 @@ export function regexBasedFormatter(re, guesstimateTypeFn = getGuesstimateType,
     _numbers(text) { return _.chunk(text.match(re).slice(1), 2).map(([num, suffix]) => parseNumber(num, suffix)) },
   }
 }
+
+
+
+
+
+// The function: shorthandIntoLognormalFormattingStep 
+// is used in foretold/packages/guesstimator/formatter/formatters/Function.js
+// In order to be able to format inputs like "1 to 100" when inside a multimodal, like "=mm(normal(10,5), 1 to 100))".
+
+// The function: shorthandIntoLognormalReplacer
+// is inside shorthandIntoLognormalFormatting. This might be marginally slower, but it makes clear that it is not used by any other function.
```

Comment:
I think this is fairly obvious and doesn't need to be explicitly stated. I find semi-obvious comments hurt readability, because they make people less likely to see the useful comments.